dts/bindings/crypto/nordic,nrf-ccm.yaml

```diff
@@ -20,3 +20,10 @@ properties:
       Indicates whether the CCM peripheral supports the extended length
       (8 bits) of the LENGTH field in encrypted/decrypted packets.
       If not set, only the default length (5 bits) is supported.
+
+  headermask:
```

Comment:
I think that `headermask-supported` will work better and it will be consistent with similar properties that we have for RADIO (`ble-2mbps-supported`, `ble-coded-phy-supported`, etc.).

Reply:
Will update.